### ISSUE TITLE
Conflicting Intel Driver & Support Assistant packages

### ISSUE THREAD

**Issue by me-and** [microsoft/winget-pkgs#1742]
There are two manifests in this repository for the Intel Driver & Support Assistant, that differ only by the capitalisation of "a" in the "and". This seems unnecessary and makes installing one or the other unnecessarily difficult, as you need to (a) spot that the distinction is the capitalisation and (b) go hunting to remind yourself what the syntax is for the case-sensitive selection. One or the other should probably be deleted, or possibly they should be merged.

As best I can tell, this is entirely accidental. One was added in #1165 by @NathanBnm, the other in #1214 by @Reset12138. 

I've no particular preference about which is preserved and which is maintained; the one from #1214 (IntelDriverandSupportAssistant, i.e. without the capital "A") specifies a more recent version, but as best I can tell they're both downloading the same installer. One has the argument `/silent`, the other `/S /NORESTART`, but that, the version and the installer hash seem to be the only substantive differences between the manifests.

**Comment by NathanBnm** [comment]
I was the first to submit a PR, the second PR shouldn't have been merged. I even created a PR to add a notice about duplicates in the README to avoid this case.

Every word should be capitalized in the identifier so the correct identifier is the one with "And".

Moreover using `/silent` as silent argument does not work (I tried myself). The required silent arguments are `/S /NORESTART`.

I partially based my PR on the chocolately package to use the same identifier and app moniker.
